src/Framework/Sources/JavaScript/Background/chrome/storage.js

```diff
@@ -64,7 +80,10 @@ module.exports = {
     },
     onChanged: {
         addListener(callback) {
-            onChangeListeners.push(callback);
+            changeEmitter.on('storage', callback);
         },
-    },
+        removeListener(callback) {
+            changeEmitter.off('storage', callback);
```

Comment:
There is no removeListener in Chrome docs https://developer.chrome.com/extensions/storage#event-onChanged

Reply:
yes, google doesn't mention it explicitly, unfortunatelly, but each addListener always has its removeListener counterpart